server/src/models/user.ts

```diff
@@ -0,0 +1,52 @@
+import { db } from "../db";
+
+export class User {
+  static async login(id: string, username: string): Promise<string> {
```

Comment:
```suggestion
  static async loginOrRegister(id: string, username: string): Promise<string> {
```
this naming might be more reflective of what's happening in the function

Reply:
Good point, will make the change!